.ci_support/linux_64_.yaml

```diff
@@ -17,17 +17,17 @@ cxx_compiler:
 cxx_compiler_version:
 - '13'
 docker_image:
-- quay.io/condaforge/linux-anvil-cos7-x86_64
+- quay.io/condaforge/linux-anvil-x86_64:alma9
 libcurl:
 - '8'
 libtiff:
-- '4.6'
+- '4.7'
+proj:
+- '9.5'
```

Comment:
It's odd that `proj` is getting pins for itself.  @conda-forge/proj-4, do you all think this is a bug? If so, should we report it to conda-smithy or somewhere else?

Reply:
@conda-forge/core did something change? This is probably not an issue but it is odd.